translations/de.log

```diff
@@ -0,0 +1,97 @@
+orig->feedrate: 'Feedrate ', 'Feedrate '
+orig->confirmFactoryReset: 'Confirm factory reset', 'Alle Einstellungen zurücksetzen'
+orig->confirmFileDelete: 'Confirm file delete', 'Die Datei wird gelöscht'
+orig->areYouSure: 'Are you sure?', 'Sind sie sicher?'
+orig->touchTheSpot: 'Touch the spot', 'Bitte auf den Punkt tippen'
+orig->moveHead: 'Move head', 'Kopf bewegen'
+orig->extrusionAmount: 'Extrusion amount (mm)', 'Extrusionsmenge (mm)'
+orig->extrusionSpeed: 'Speed (mm/s)', 'Geschwindigkeit (mm/s)'
+orig->extrude: 'Extrude', 'Extrudieren'
+orig->retract: 'Retract', 'Zurückziehen'
+orig->babyStepping: 'Baby stepping', 'Einzelschritte'
+orig->currentZoffset: 'Current Z offset: ', 'Aktueller Z-Versatz: '
+orig->message: 'Message', 'Nachricht'
+orig->messages: 'Messages', 'Nachrichten'
+orig->firmwareVersion: 'Panel Due firmware version ', 'Panel Due Firmwareversion '
+orig->response: 'Response', 'Antwort'
+orig->filesOnCard: 'Files on card ', 'Dateien auf Karte '
+orig->macros: 'Macros', 'Makros'
+orig->error: 'Error ', 'Fehler '
+orig->accessingSdCard: ' accessing SD card', ' beim Zugriff auf SD-Karte'
+orig->fileName: 'Filename: ', 'Dateiname: '
+orig->fileSize: 'Size: ', 'Größe: '
+orig->layerHeight: 'Layer height: ', 'Schichthöhe: '
+orig->objectHeight: 'Object height: ', 'Objekthöhe: '
+orig->filamentNeeded: 'Filament needed: ', 'Benötigtes Filament: '
+orig->generatedBy: 'Sliced by: ', 'Erzeugt mit: '
+orig->lastModified: 'Last modified: ', 'Letzte Änderung: '
+orig->estimatedPrintTime: 'Estimated print time: ', 'Geschätzte Druckdauer: '
+orig->simulatedPrintTime: 'Simulated print time: ', 'Errechnete Druckdauer: '
+orig->simulate: 'Simulate', 'Simulieren'
+Status Values:
+orig->statusValues[(unsigned int)OM::PrinterStatus::connecting]: 'Connecting', 'Verbinde'
+orig->statusValues[(unsigned int)OM::PrinterStatus::idle]: 'Idle', 'Leerlauf'
+orig->statusValues[(unsigned int)OM::PrinterStatus::printing]: 'Printing', 'Druckt'
+orig->statusValues[(unsigned int)OM::PrinterStatus::stopped]: 'Halted', 'Angehalten'
+orig->statusValues[(unsigned int)OM::PrinterStatus::configuring]: 'Starting up', 'Starte'
+orig->statusValues[(unsigned int)OM::PrinterStatus::paused]: 'Paused', 'Pausiert'
+orig->statusValues[(unsigned int)OM::PrinterStatus::resuming]: 'Resuming', 'Fortsetzen'
+orig->statusValues[(unsigned int)OM::PrinterStatus::flashing]: 'Firmware upload', 'Firmware-Upload'
+orig->statusValues[(unsigned int)OM::PrinterStatus::toolChange]: 'Changing tool', 'Wechsle Tool'
+orig->statusValues[(unsigned int)OM::PrinterStatus::simulating]: 'Simulating', 'Simuliert'
+orig->statusValues[(unsigned int)OM::PrinterStatus::off]: 'Off', 'Aus'
+orig->statusValues[(unsigned int)OM::PrinterStatus::cancelling]: 'Cancelling', MISSING
+Colour Scheme Names:
+orig->colourSchemeNames[j]: 'Light theme', 'Anzeige hell'
+orig->colourSchemeNames[j]: 'Dark theme 1', 'Anzeige inv. 1'
+orig->colourSchemeNames[j]: 'Dark theme 2', 'Anzeige inv. 2'
```

Comment:
"Anzeige dunkel 2"
If it is to long, leave it like it is.

Reply:
Done.